kouta-backend/src/test/scala/fi/oph/kouta/integration/HakukohdeSpec.scala

```diff
@@ -126,7 +126,7 @@ class HakukohdeSpec extends UnitSpec with KoutaIntegrationSpec with AccessContro
     val oid = put(uusiHk)
```

Comment:
Tässä `put` issa vois käyttää varmaan `ophSession` ia ja vaihtaa takaisin tuolta tuon `isMuokkaajaOphVirkailija = Some(true)`, niin testissä tehtäis sitä, mitä testin kuvauksessa sanotaan?

Reply:
Tai sitten poistaa koko testin, koska noissa jälkimmäisissä testeissä tulee kyllä sama asia testattua.

Reply:
...tai sit muuttaa tämän testin kuvausta ja tehä `put` ei-ophSessionilla, niin tulee testattua tää false-arvon asettaminen myös.